Sunrise/Models/Money.swift

```diff
@@ -9,7 +9,7 @@ struct Money: Mappable {
     // MARK: - Properties
 
     var currencyCode: String?
-    var centAmount: Int?
+    var centAmount: Double?
```

Comment:
`return currencyFormatter.stringFromNumber(centAmount / 100) ?? ""` Cents weren't properly calculated, but now the tests are covering that as well. If we'll always have Int value for this one, we can just divide by 100.0.